learn/configuration/instance_options.md

```diff
@@ -4,10 +4,99 @@ sidebarDepth: 2
 
 # Configure Meilisearch at launch
 
-You can configure Meilisearch at launch with **environment variables** and **command-line options**.
+You can configure Meilisearch at launch with **command-line options**, **environment variables**, or a **configuration file**.
 
 These startup options affect your entire Meilisearch instance, not just a single index. For settings that affect search within a single index, see [index settings](/learn/configuration/settings.md).
 
+## Configuration file
+
+Meilisearch accepts a configuration file in the `.toml` format as an alternative to command-line options and environment variables. Configuration files can be easily shared and versioned, and allow you to define multiple options.
+
+**If used simultaneously, environment variables can overwrite the configuration file, and command-line options can overwrite environment variables.**
+
+You can download the configuration file using the following command:
+
+<CodeSamples id="instance_options_config_file_download_1" />
+
+This default configuration file is divided into sections separated by comments. Each key has a brief description followed by a link to the relevant section of the documentation. Some keys are commented out as they are not required when launching Meilisearch, for example, `import_snapshot`, or because the value provided in the file is not the default value like `max_indexing_memory`.
+
+By default, Meilisearch will use the file `./config.toml` in the working directory as the default configuration file. You will see it when you launch Meilisearch:
```

Comment:
I think we can drop the article. Also, we don't need to use both `./` and working directory, in my opinion because the working directory is `./`.

```suggestion
By default, Meilisearch will use the `config.toml` in the working directory as the default configuration file. You will see it when you launch Meilisearch:
```